homeassistant/components/epson/media_player.py

```diff
@@ -179,6 +219,7 @@ async def async_turn_off(self):
         """Turn off epson."""
         if self._state == STATE_ON:
             await self._projector.send_command(TURN_OFF)
+            self._state = STATE_OFF
```

Comment:
Why add this here? And why not add it to `async_turn_off` too? 

Since this is a polling entity, HA will fetch the state from the projector right after it calls this command and this will be overwritten.